tone-analyzer/v3.js

```diff
@@ -83,4 +83,26 @@ ToneAnalyzerV3.prototype.tone = function(params, callback) {
   return requestFactory(parameters, callback);
 };
 
+ToneAnalyzerV3.prototype.tone_chat = function(params, callback) {
+  if (!params || !params.utterances) {
+    callback(new Error('Missing required parameters: utterances'));
```

Comment:
This will be handled automatically if you set `requiredParams` and `originalParams` on the the `parameters` object. See https://github.com/watson-developer-cloud/node-sdk/blob/master/speech-to-text/v1.js#L818 for an example